.github/actions/breeze/action.yml

```diff
@@ -25,12 +25,6 @@ outputs:
 runs:
   using: "composite"
   steps:
-    - name: "Setup python"
```

Comment:
I am afraid this will not work if we want to merge it and would like to freely switch between the "public", "AWS" and "ARC" runners. 

I would  prefer to have the  case where we have a way to easily switch to choose which runners we are using on a PR-by PR basis (i.e. for example apply label to make specific PR to run on ARC - similalry as we do with "use public runners") - at least for a few week transition period.

I think what can be done is that this action can be conditional based on (yes, you guessed it) selective checks output. I think when you are calling a composite action, you are able to pass it a parameter from the calling action, so we could pass "use-arc" or simlar selective check output and based on that we could either run the setup python action or not.